### PR TITLE
feat(backend): Domain Information Update for admins

### PR DESCRIPTION
**Pull request by Quekyy**

### CODE REVIEW

File: apps/backend/src/routes/home/+page.svelte [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+  import { domainStore, authStore } from '$lib/stores';
+  import { fade } from 'svelte/transition';
+  import * as yup from 'yup';
 
   // State variables
   let showInfoCard = false;
   let selfDestructModal: HTMLDialogElement;
   let fileAccessModal: HTMLDialogElement;
+  let LogoutModal: HTMLDialogElement;
+  let SettingsModal: HTMLDialogElement;
```
Comment (taronaeo) [inline apps/backend/src/routes/home/+page.svelte:19]: Only classes, interfaces, and types use TitleCase. Otherwise, please use camelCase

**Review by taronaeo — CHANGES_REQUESTED** [review]
One small change

File: apps/backend/src/routes/home/+page.svelte (hunk 1 above)
Comment (Quekyy) [reply apps/backend/src/routes/home/+page.svelte]: Fixed, changed to
logoutModal and settingsModal

**Review by taronaeo — APPROVED** [review]
LGTM

**Pull request merged**